### DISCUSSION TITLE
Best suited Algorithm which can be taken into measure for ML model

### DISCUSSION THREAD

**Discussion by thelordbuddhaclub** [thelordbuddhaclub/Rak_sh_ak#18] [Q&A] [answered]
Which Machine Learning algorithm can be used to build the model for disease detection ?

**Comment by PIYUSH-MISHRA-00** [comment, marked as answer]
Considering many approaches like SVM, DT, NB, and KNN. RF algorithm can be used to build the model as this approach can be used without the hyper parameter tuning of the data and even a great result is achieved most of the time.
